lua/neorg/modules/core/summary/module.lua

```diff
@@ -363,7 +363,11 @@ module.events.subscribed = {
 
 module.on_event = function(event)
     if event.type == "core.neorgcmd.events.summary.summarize" then
-        module.public.generate_workspace_summary(event.buffer, event.cursor_position, event.content)
+	-- Extract any additional arguments past the first
+	-- So that :Neorg generate_workspace_summary a b c
+	-- Will result in include_categories = {"a", "b", "c"}
+	local include_categories = {unpack(event.content)}
+        module.public.generate_workspace_summary(event.buffer, event.cursor_position, include_categories)
```

Comment:
Let's just apply this suggestion so the comment is more accurate and the formatting matches the rest of the repo!

```suggestion
        -- Remove `data` key, and take only the numerical keys from event.content
        -- these numerical keys are the category args passed to the command
        local include_categories = { unpack(event.content) }
        module.public.generate_workspace_summary(event.buffer, event.cursor_position, include_categories)
```